### ISSUE TITLE
No types for InlineKeyboardMarkup, InlineKeyboardButton and all structs

### ISSUE THREAD

**Issue by freemin7** [Arkoniak/Telegram.jl#24]
As far as i can see the current API scrapping only scraped the functions but not the struct types. As such there is no type for InlineKeyboardMarkup et al.

**Comment by Arkoniak** [comment]
Yes, it's much harder to automatically map telegram types to the Julia types.
But some most common structs can be implemented manually, I presume. I'll be glad to accept corresponding pull requests.

**Comment by freemin7** [comment]
So you would prefer a manual implementation would implementing it using JSON3 and structtypes be fine?
